lib/pbench/agent/tool_data_sink.py

```diff
@@ -165,14 +192,10 @@ def launch(self):
                 )
             return 0
 
+        #TODO: USE WITH
         config = open("prometheus.yml", "w")
 
-        config.write("global:\n  scrape_interval: 1s\n  evaluation_interval: 1s\n\n")
-        # config.write("alerting:\n  alertmanagers:\n  - static_configs:\n    - targets:\n\nrule_files:\n\n")
-        config.write(
-            "scrape_configs:\n  - job_name: 'prometheus'\n    static_configs:\n    - targets: ['localhost:9090']\n\n"
-        )
-
+        tool_context = []
         for host in self.host_tools_dict:
             if host.startswith("local"):
```

Comment:
Is this supposed to distinguish between "localhost" and everybody else? Or is there a possibility of some other host having a name that starts wtih "local"? If the former, you should compare it with "localhost", not just checking a prefix. If the latter, maybe a comment is indicated here - otherwise, I need to remember what host_tools_dict contains (which I probably should, but I don't :-) ).

Reply:
Yeah, this was in order to correct for the case where a user's formal hostname is something like "localhost.localdomain" on an unnamed vm. In order to still be usable, we would need ip as hostname. Not sure if we support this system-wide, though, but for now the support here doesn't hurt anything, will add a comment